ui/Screen/NetworkDiagnostics/ConnectedPeers.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="typescript">
+  import { StatusType } from "../../src/localPeer.ts";
+  import { store } from "../../src/screen/network";
+
+  import Remote from "../../DesignSystem/Component/Remote.svelte";
+
+</script>
+
+<style>
+  table {
+    table-layout: fixed;
+    width: 100%;
+    border-collapse: collapse;
+    border: 3px solid purple;
+  }
+
+</style>
+
+<Remote {store} let:data={{ status }}>
+  <div>
+    {#if status.type === StatusType.Online}
```

Comment:
As mentioned in in https://github.com/radicle-dev/radicle-upstream/issues/1798 `Remote` disables type checking. `status` will be of type any and errors in accessing it’s fields will not be caught. I suggest we use a simple `if` statement. It has the same functionality but does type checking.

```suggestion
{#if $store.status === remote.Status.Success}
  <div>
    {#if $store.data.type === StatusType.Online}
```